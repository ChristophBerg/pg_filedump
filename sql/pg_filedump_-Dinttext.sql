\! pg_filedump -D int,text testfile | sed -e 's/recoff 0x......../recoff 0x......../'
