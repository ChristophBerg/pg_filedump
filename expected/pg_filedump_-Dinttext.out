\! pg_filedump -D int,text testfile | sed -e 's/recoff 0x......../recoff 0x......../'

*******************************************************************
* PostgreSQL File/Block Formatted Dump Utility
*
* File: testfile
* Options used: -D int,text
*******************************************************************

Block    0 ********************************************************
<Header> -----
 Block Offset: 0x00000000         Offsets: Lower      36 (0x0024)
 Block: Size 8192  Version    4            Upper    8088 (0x1f98)
 LSN:  logid      0 recoff 0x........      Special  8192 (0x2000)
 Items:    3                      Free Space: 8052
 Checksum: 0x0000  Prune XID: 0x00000000  Flags: 0x0004 (ALL_VISIBLE)
 Length (including item array): 36

<Data> -----
 Item   1 -- Length:   32  Offset: 8160 (0x1fe0)  Flags: NORMAL
COPY: 1	one
 Item   2 -- Length:   32  Offset: 8128 (0x1fc0)  Flags: NORMAL
COPY: 2	two
 Item   3 -- Length:   34  Offset: 8088 (0x1f98)  Flags: NORMAL
COPY: 3	three


*** End of File Encountered. Last Block Read: 0 ***
